save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -252,32 +252,29 @@ class LnkUserOrganizationController(
         lnkUserOrganizationService.getSuperOrganizationsWithRole((authentication.details as AuthenticationDetails).id)
     )
 
-    @GetMapping("/by-user")
+    @PostMapping("/by-user-and-filters")
```

Comment:
There's no user in the request parameters any longer, so `/by-user-and-filters` is misleading.

Additionally, your REST API should also be memorable by humans, so I'd rather not use two words when one is enough. Having said the above, `/filtered` is better than `/by-filters` because the latter may be easily mistaken for `/by_filters` and `/byFilters`, and so on.

@nulls, WDYT?

Reply:
> Additionally, your REST API should also be memorable by humans, so I'd rather not use two words when one is enough. Having said the above, /filtered is better than /by-filters because the latter may be easily mistaken for /by_filters and /byFilters, and so on.

`by-filter` is more human readable for me that `/filtered`

Reply:
> `/by-filter` is more human readable for me that `/filtered`

Ok, let's have it named `/by-filter` then.